cmd/stackmargin/src/lib.rs

```diff
@@ -72,6 +80,12 @@ fn stackmargin(context: &mut humility::ExecutionContext) -> Result<()> {
     };
 
     for i in 0..size {
+        if let Some(HubrisTask::Task(ndx)) = dump_task {
+            if ndx != i {
```

Comment:
Same here, a comment would be helpful (I'm guessing this is to avoid reading the extra memories mapped to the dump task, but not sure).